.github/workflows/pr-realm-js.yml

```diff
@@ -698,11 +702,32 @@ jobs:
             ~/.android/adb*
           key: avd-29
 
+      - name: Bundle test app
+        working-directory: integration-tests/environments/react-native-test-app
+        run: |
+          mkdir dist
+          npx react-native bundle --entry-file index.js --platform android --dev false --bundle-output dist/main.android.jsbundle --assets-dest dist/res
+
+      - name: Generate a keystore for signing
+        working-directory: integration-tests/environments/react-native-test-app
+        env:
+          STORE_PATH: debug.keystore
+          STORE_PASS: android
+          KEY_ALIAS: debug-key
+          KEY_PASS: android
+        run: |
+          keytool -genkey -v -keyalg RSA -keysize 2048 -validity 10000 -dname cn=Realm -keystore ${{ env.STORE_PATH }} -storepass ${{ env.STORE_PASS }} -alias ${{ env.KEY_ALIAS }} -keypass ${{ env.KEY_PASS }}
+          jq --arg storeFile ${{ env.STORE_PATH }} --arg storePassword ${{ env.STORE_PASS }} --arg keyAlias ${{ env.KEY_ALIAS }} --arg keyPassword ${{ env.KEY_PASS }} '. +{android: {signingConfigs: { release: { $storeFile, $storePassword, $keyAlias, $keyPassword } }}}' app.json > patched-app.json
+          mv patched-app.json app.json
```

Comment:
This generates a temporary keystore and key to sign the release APK and patch the RNTA app.json to pick it up when building the app.